definitions/defaults/actions.lua

```diff
@@ -2,601 +2,676 @@ local lib = require('library')
+  UnsoloAllItems = 41185,
+  ToggleAutomaticRecordArm = 40740,
+  ToggleStopAtEndOfTimeSelectionIfNoRepeat = 41834,
+  ToggleViewMixer = 40078,
+  ToggleVolumeEnvelope = 40406,
+  SelectWidthEnvelope = 41870,
+  ToggleTakeMuteEnvelope = 40695,
+  ToggleTakePanEnvelope = 40694,
+  ToggleTakePitchEnvelope = 41612,
+  ToggleTakeVolumeEnvelope = 40693,
+  SetTrackInputToMatchFirstSelected = "_SWS_INPUTMATCH",
+  TrackToggleSendToParent = "_SWS_TOGMPSEND",
+  TrackWithNumber = custom.move.trackWithNumber,
+  TrimItemLeftEdge = 41305,
+  TrimItemRightEdge = 41311,
+  TrimSelectedNoteLeftEdgeToEditCursor = 40790,
+  TrimSelectedNoteRightEdgeToEditCursor = 40791,
+  UnarmAllEnvelopes = 41163,
+  UnarmSelectedTracks = "_XENAKIOS_SELTRAX_RECUNARMED",
+  UncollapseFolder = "_SWS_UNCOLLAPSE",
+  Undo = {40029, prefixRepetitionCount=true},
+  UnfreezeTrack = 41644,
+  UnmuteAllTracks = 40339,
+  UnselectAllEvents = {40214, midiCommand=true},
+  UnselectEnvelopePoints = 40331,
+  UnselectItems = 40289,
+  UnselectTracks = 40297,
+  UnsoloAllTracks = 40340,
+  VerticalScrollEnd = "_XENAKIOS_TVPAGEEND",
+  VerticalScrollStart = "_XENAKIOS_TVPAGEHOME",
+  ToggleShowInputFxChain = 40844,
+  ViewFxChainMaster = 40846,
+  ZoomAllItems = {"SaveItemSelection", "SelectAllItems", "ZoomItemSelection", "RestoreItemSelection"},
+  ZoomAllTracks = {"SaveTrackSelection", "SelectAllTracks", "ZoomTrackSelection", "RestoreTrackSelection"},
+  ZoomInHoriz = {1012, prefixRepetitionCount=true},
+  ZoomInVert = {40111, prefixRepetitionCount=true},
+  ZoomItemSelection = "_SWS_HZOOMITEMS",
+  ZoomOutHoriz = {1011, prefixRepetitionCount=true},
+  ZoomOutVert = {40112, prefixRepetitionCount=true},
+  ZoomProjectTimeline = 40295,
+  ZoomProject = {"ZoomAllTracks", "ZoomAllItems", "ScrollToSelectedTracks"},
+  ZoomRedo = {"_SWS_REDOZOOM", prefixRepetitionCount=true},
+  ZoomTimeAndTrackSelection = {"ZoomTrackSelection", "ZoomTimeSelection"},
+  ZoomTimeSelection = 40031,
+  ZoomTrackSelection = "_SWS_VZOOMFITMIN",
+  ZoomUndo = {"_SWS_UNDOZOOM", prefixRepetitionCount=true},
+  NudgeTrackVolumeDown = 40116, -- 0.05 dB
+  PlayAndSkipTimeSelection = 40317,
+
+  RouteUpdate = lib.routing.updateState,
+  RouteLogSelection = lib.routing.logRoutingInfoForSelectedTracks,
+  RouteRemove = lib.routing.remove,
+  RouteRemoveAllSends = lib.routing.removeAllSends,
+  RouteRemoveAllRecieves = lib.routing.removeAllRecieves,
+  RouteTestCodedT = lib.routing.testCodedTargets,
+
+  VKBSendMidi_A_2 = lib.midi.sendMidiNote_A_2,
+  VKBSendMidi_Gs2 = lib.midi.sendMidiNote_Gs2,
+  VKBSendMidi_G_2 = lib.midi.sendMidiNote_G_2,
+  VKBSendMidi_Fs2 = lib.midi.sendMidiNote_Fs2,
+  VKBSendMidi_F_2 = lib.midi.sendMidiNote_F_2,
+  VKBSendMidi_E_2 = lib.midi.sendMidiNote_E_2,
+  VKBSendMidi_Ds2 = lib.midi.sendMidiNote_Ds2,
+  VKBSendMidi_D_2 = lib.midi.sendMidiNote_D_2,
+  VKBSendMidi_Cs2 = lib.midi.sendMidiNote_Cs2,
+  VKBSendMidi_C_2 = lib.midi.sendMidiNote_C_2,
+
+  SelectAllItemsInCurrentTimeSel = 40717,
```

Comment:
This section is very different from all the rest above, and it's best to keep things consistent when possible